tools/shiki-twoslash/src/client/styles/twoslash.scss

```diff
@@ -0,0 +1,378 @@
+/* stylelint-disable selector-class-pattern */
```

Comment:
Files like this may be updated with sass nesting so that it can have better readability